packages/gis/lua/Layer.lua

```diff
@@ -534,51 +545,52 @@ Layer_ = {
 	-- from the reference layer. It sums the intersection areas of the object with all the polygons
 	-- of the reference layer. Because of that, if there is some overlay between the polygons of the
 	-- reference layer, it might create attribute values greater than one.
-	-- & attribute, layer & split \
+	-- & attribute, layer & progress, split \
 	-- "average" & Average of quantitative values from the objects that have some intersection
 	-- with the cell, without taking into account their geometric properties. When using argument
 	-- area, it computes the average weighted by the proportions of the respective intersection areas.
 	-- Useful to distribute atributes that represent averages, such as per capita income.
-	-- & attribute, layer, select  & area, missing, band, dummy, pixel, split  \
+	-- & attribute, layer, select  & area, missing, band, dummy, pixel, progress, split  \
 	-- "count" & Number of objects that have some overlay with the cell.
-	-- & attribute, layer & dummy, pixel, split \
-	-- "coverage" & Percentage of each qualitative value covering the cell, using polygons or
+	-- & attribute, layer & dummy, pixel, progress, split \
+	-- "coverage" & Percentage or total area of each qualitative value covering the cell, using polygons or
 	-- raster data. It creates one new attribute for each available value, in the form
 	-- attribute.."_"..value, where attribute is the value passed as argument to fill and
 	-- value represent the different values in the input data.
 	-- The sum of the created attribute values for a given cell will range
-	-- from zero to one, according to the area of the cell covered by pixels.
+	-- from zero to one (default) or the total area (area = true), according to the area of the
```

Comment:
from zero to one indicating the percentage of the cell covered by pixels of each class (default), or the total area covered by each class, according to the area of a pixel in the raster data (when using area = true).